### PR TITLE
Fix ring sensor configuration

### PR DESCRIPTION
**Pull request by cgtobi**
**Description:**
Fix ring configuration

Related to #6385.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.ring.markdown [hunk 1]
```diff
@@ -25,15 +25,25 @@ sensor:
...
-  - **battery**: Return the battery level from device
-  - **last_activity**: Return the timestamp from the last event captured (ding/motion/on demand) by the Ring doorbell camera
-  - **last_ding**: Return the timestamp from the last time the Ring doorbell button was pressed
-  - **last_motion**: Return the timestamp from the last motion event captured by the Ring doorbell camera
-  - **volume**: Return the volume level from the device.
-  - **wifi_signal_category**: Return the WiFi signal level from the device.
-  - **wifi_signal_strength**: Return the WiFi signal strength (dBm) from the device.
+{% configuration %}
+monitored_conditions:
+  type: array
```
Comment (frenck) [inline source/_components/sensor.ring.markdown]: `array` is not a valid type for the documentation. Please use `list`.

**Review by frenck — CHANGES_REQUESTED** [review]
@cgtobi Thanks! Left a small comment, could you please take a look? 👍

File: source/_components/sensor.ring.markdown (hunk 1 above)
Comment (cgtobi) [reply source/_components/sensor.ring.markdown]: Ah, should have noticed that.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the quick update @cgtobi (and the PR of course).

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**